sky/serve/autoscalers.py

```diff
@@ -16,6 +16,9 @@
 
 logger = sky_logging.init_logger(__name__)
 
+_UPSCALE_DELAY_S = 300
```

Comment:
Should we expose this option to user?

Reply:
current `AUTOSCALER_SCALE_FREQUENCY_SECONDS` is not exposed to the user